<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadFrame</class>
 <widget class="QWidget" name="DownloadFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateTimeEdit" name="startDateTime">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelEnd">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="endDateTime">
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelCamera">
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="camera">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="browse">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="path"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelSize">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="size">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="status">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
